do-not-commit/is-not-is-not.py

```diff
@@ -0,0 +1,13 @@
+x = "foo"
+
+# ruleid: is-not-is-not
+if x is (not "hello there"):
```

Comment:
Found string comparison using 'is' operator. The 'is' operator
is for reference equality, not value equality, and therefore should
not be used to compare strings. For more information, see
https://github.com/satwikkansal/wtfpython#-how-not-to-use-is-operator" ([.common-mistakes.is-comparison-string.string-is-comparison](https://semgrep.dev/r/.common-mistakes.is-comparison-string.string-is-comparison))
#
<div>
<a href="https://semgrep.dev">
<img align="right" src="https://r2c.dev/semgrep_icon_text.svg" height=28px alt="" />
</a>

<sub> [How do I suppress this?](https://semgrep.dev/docs/running-rules/#managing-findings) **·** [Report a bug](https://github.com/returntocorp/semgrep-rules/issues/new?assignees=&labels=&template=rule_bug_report.md&title=.common-mistakes.is-comparison-string.string-is-comparison) **·** [Manage rule](https://semgrep.dev/manage/projects)</sub>
</div>